dev/bench/bench.sh

```diff
@@ -346,6 +346,8 @@ create_opam() {
 
     echo "$1_coq_commit_long = $COQ_HASH_LONG"
 
+    opam install coq-native
```

Comment:
Is this testing what you want? As far as I can see you will mainly test how fast OCaml compiles files, except for the developments that call native compute explicitly, and this we don't know well I think.

Reply:
We also care about how fast the compilation goes so that's fine imo